### PR TITLE
Orderconfirmation

### PR DESCRIPTION
**Pull request by saikrishna9541**
### What issues does this PR fix or reference?
#<Insert GitHub Issue>, @<Insert GUS WI>@

### Functionality Before
<insert gif and/or summary>

### Functionality After
<insert gif and/or summary>

### How to Test/Testing Effort 
<insert gif and/or summary>

### CODE REVIEW

File: examples/b2c/checkout/order-confirmation-email-flow/email/unfiled$public/B2B2C_Order_Confirmation_Email.email [hunk 1]
```diff
@@ -9,25 +9,26 @@
...
         </div>
          <div style="padding: 0px 20px 20px;">
-            <h1  style="font-size:18px; font-weight:bold; padding:0px; margin: 0px;">Thank you for your order, {{{OrderSummary.OCE_Account_First_Name__c}}}!</h1>
-            <p style="padding:10px 0px; margin: 0px;">Please find your order details below.</p>
-            <h2 style="font-size:14px; font-weight:bold; padding:0px; margin: 0px;">Confirmation Number: {{{OrderSummary.OrderNumber}}}</h2>
+            <h1  style="font-size:20px; font-weight:bold; padding:0px; margin: 0px;">Thank you for your order, {{{OrderSummary.OCE_Account_First_Name__c}}}!</h1>
+            <p style="font-size:16px;padding:10px 0px; margin: 0px;">Please find your order details below.</p>
+            <h2 style="font-size:16px; font-weight:bold; padding:0px; margin: 0px;">Confirmation Number: {{{OrderSummary.OrderNumber}}}</h2>
+            <a style="display: block;background-color:#000;color:#ffffff;font-size:13px;font-weight:400;line-height:20px;padding:6px;text-align:center;text-decoration: none;margin-top:10px;" href={{{OrderSummary.OCE_Order_Url__c}}} title="View Order Status">View Order Status</a>
```
Comment (nkurapati) [inline examples/b2c/checkout/order-confirmation-email-flow/email/unfiled$public/B2B2C_Order_Confirmation_Email.email:15]: nil: Please add spaces in styles all the places.

File: examples/b2c/checkout/order-confirmation-email-flow/email/unfiled$public/B2B2C_Order_Confirmation_Email.email [hunk 2]
```diff
@@ -9,25 +9,26 @@
...
          <hr style="border: 1px solid #CDCDCD; margin:0px;"/>
          <div style="padding:20px 10px 0px;">
             <table width="100%">
                 <tr style="vertical-align:top;">
                     <td width="50%">
                         <div style="padding: 0px 10px;">
-                        <h2 style="font-size:14px; font-weight:bold; padding:0px; margin: 0px;">Shipping Address</h2>
-                        <p style="padding:10px 0px; margin: 0px;">{{{OrderSummary.OCE_Shipping_Address__c}}}</p>
+                        <h2 style="font-size:16px; font-weight:bold; padding:0px; margin: 0px;">Shipping Address</h2>
+                        <p style="font-size:16px; padding:10px 0px; margin: 0px;">{{{OrderSummary.OCE_Shipping_Address__c}}}</p>
```
Comment (nkurapati) [inline examples/b2c/checkout/order-confirmation-email-flow/email/unfiled$public/B2B2C_Order_Confirmation_Email.email:24]: Header & context with same font size?  Is it same in design?

File: examples/b2c/checkout/order-confirmation-email-flow/flows/B2B2C_Order_Confirmation_Email_Notification_Flow.flow [hunk 3]
```diff
@@ -95,21 +107,23 @@
             <assignToReference>orderItems</assignToReference>
             <operator>Add</operator>
             <value>
-                <stringValue>&lt;/td&gt;&lt;td&gt;$</stringValue>
-            </value>
-        </assignmentItems>
-        <assignmentItems>
```
Comment (nkurapati) [inline examples/b2c/checkout/order-confirmation-email-flow/flows/B2B2C_Order_Confirmation_Email_Notification_Flow.flow:101]: Why is it removed?

Comment (saikrishna9541) [reply examples/b2c/checkout/order-confirmation-email-flow/flows/B2B2C_Order_Confirmation_Email_Notification_Flow.flow:101]: These are of total amount string values. To the UX https://www.figma.com/file/fbCXLQCt9PMk1aIrJxIXEe/246-Order-Confirmation-Email?type=design&node-id=783-23505&t=64Yey6WX1mt7ZwnV-0, I have removed it and kept only each product unit value.

File: examples/b2c/checkout/order-confirmation-email-flow/email/unfiled$public/B2B2C_Order_Confirmation_Email.email (hunk 2 above)
Comment (saikrishna9541) [reply examples/b2c/checkout/order-confirmation-email-flow/email/unfiled$public/B2B2C_Order_Confirmation_Email.email:24]: YEs, just the font weight difference only.

**Comment by tarcang** [comment]
@saikrishna9541 What is the state of this PR? Is it still in draft?

**Pull request closed without merging**